ucm/sof-sdw-rt711-1308-715/HiFi

```diff
@@ -0,0 +1,142 @@
+# Use case Configuration for skl-hda-card
+
+SectionVerb {
+
+	EnableSequence [
+		cdev "hw:sofsdwrt7111308"
```

Comment:
wow. that looks bad. we probably need to change the card name in the machine driver.

Reply:
> wow. that looks bad. we probably need to change the card name in the machine driver.

@plbossart Sorry, I didn't get your point. Do you think the card name is too long or?

Reply:
yes, there's no space for the rt715...

Reply:
@plbossart It is hard to show all 3 codecs' name on the card name. One option is to remove the "sdw" from the card name. i.e. Card name will be sofrt7111308715. Or we can just remove the codec name at all. e.g. sofrtksdwcodec. What do you think?

Reply:
I am tempted to go for an encoding that gives a hint about the topology, e.g.
sdw-rt-111 -> 1 rt711, 1 rt1308, 1 rt715
sdw-rt-121 -> 1 rt711, 2 rt1308, 1 rt715

That would make it easier to maintain UCM files.